tsconfig.json

```diff
@@ -4,6 +4,8 @@
         "module": "commonjs",
         "strict": true,
         "sourceMap": true,
+        "outDir": "dist",
+        "declaration": true,
```

Comment:
The outdir breaks the maps and debugging when using webpack. Then without it the declaration files pollute the source (to be fair, they shouldn't be created anyway as you have the types file).
```suggestion
```